lib/cli/cs/relate.rb

```diff
@@ -44,7 +44,7 @@
           result    = template.result(binding)
 
           # cache result and filename
-          filename        = "#{data[:from]}_#{data[:to]}".gsub(/ /, '')
+          filename        = "#{data[:from]}_#{data[:to]}".gsub(/ /, '').gsub('/','')
```

Comment:
Should be combined: `.gsub(/(\s|\/)/, '')` (always prefer single gsub to chained when possible)